deploy/kubernetes/base/controller.yaml

```diff
@@ -5,6 +5,7 @@ kind: Deployment
 apiVersion: apps/v1
 metadata:
   name: ebs-csi-controller
+  namespace: default
```

Comment:
What is the reason for explicitly adding this? By default, all resources are created in the default namespace, unless a namespace is specified.

Reply:
Yes, this is generated by template `namespace: {{ .Release.Namespace }}` in helm template. It should be exclude to kustomize profile. WIP address it.